src/index.js

```diff
@@ -1,11 +1,24 @@
-import React from "react";
-import ReactDOM from "react-dom/client";
-import "./index.css";
+import React from 'react';
```

Comment:
What linter are you using? We should all agree on one so that we do not get changes that are not really changes. Prettier is typically the recommended one for javascript

Reply:
I did not realize that my prettier plugin was not turned on. Anyhow, that is not fixed.